File: trunk/src/Servidor/Transaccional/AccesoDatos/Contrato/IDaoRol.cs
using System.Collections.Generic;
using Trascend.Bolet.ObjetosComunes.Entidades;

namespace Trascend.Bolet.AccesoDatos.Contrato
{
    public interface IDaoRol : IDaoBase<Rol, string>
    {
        IList<Rol> ObteneRolesYObjetos();
    }
}

File: src/Servidor/Transaccional/AccesoDatos_Fac/Contrato/IDaoRol.vb
Imports System.Collections.Generic
Imports Diginsoft.Bolet.ObjetosComunes.Entidades

Namespace Contrato
    Public Interface IDaoRol
        Inherits IDaoBase(Of Rol, String)
        Function ObteneRolesYObjetos() As IList(Of Rol)
    End Interface
End Namespace
